InOut/CMakeLists.txt

```diff
@@ -106,48 +94,54 @@ endif()
 
 find_package(PulseAudio)
 check_deps(USE_PULSEAUDIO PulseAudio_FOUND)
+
 if(USE_PULSEAUDIO)
     make_plugin(rtpulse rtpulse.c PulseAudio::PulseAudio)
 endif()
 
-if(PORTAUDIO_FOUND OR USE_VCPKG)
-    message(STATUS "Building rtpa module.")
-    make_plugin(rtpa rtpa.c)
-    if(USE_VCPKG)
+if(USE_PORTAUDIO)
```

Comment:
Not sure what's going on but on macOS I'm getting:
```
-- Found Portaudio: /opt/homebrew/lib/libportaudio.dylib
-- HAVE_PORTAUDIO_DEPS is disabled.
-- Found PortMidi: /opt/homebrew/lib/libportmidi.dylib
-- HAVE_PORTMIDI_DEPS is disabled.
```

with a clean checkout of this PR branch. librtpa.dylib and libpmidi.dylib are built with latest from develop. 

I see this in my CMakeCache.txt:

```
//Path to a file.
PORTAUDIO_INCLUDE_DIR:PATH=/opt/homebrew/include

//Path to a library.
PORTAUDIO_LIBRARY:FILEPATH=/opt/homebrew/lib/libportaudio.dylib

//Path to a file.
PORTMIDI_INCLUDE_DIR:PATH=/opt/homebrew/include

//Path to a library.
PORTMIDI_LIBRARY:FILEPATH=/opt/homebrew/lib/libportmidi.dylib


//Build the PortAudio I/O module
USE_PORTAUDIO:BOOL=ON

//Build the PortMIDI I/O module
USE_PORTMIDI:BOOL=ON
```

Reply:
Thanks for pointing that out. Maybe we should add a non-vcpkg build for MacOS into the CI. I don't quite understand why the variables returned are different than the `FindPortaudio.cmake`.

```
#  PORTAUDIO_FOUND - system has libportaudio
#  PORTAUDIO_INCLUDE_DIRS - the libportaudio include directory
#  PORTAUDIO_LIBRARIES - Link these to use libportaudio
```
That is the reason it's failing. I'll check it out

Reply:
Merged in #1811 to fix this issue. This old cmake find file was not really working properly.